### PR TITLE
Add summary: maas2013rectifier_1

### PR DESCRIPTION
**Pull request by kelseyball**
Summarizer: Kelsey Ball

### CODE REVIEW

File: _summaries/maas2013rectifier_1.md [hunk 1]
```diff
@@ -3,17 +3,45 @@ layout: summary
...
+author: kelseyball
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# Core idea: Rectifier DNNs > Sigmoidal DNNs in Speech Recognition
```
Comment (saikm200022) [inline _summaries/maas2013rectifier_1.md:10]: Not completely necessary here but I think it might be nice to define the DNN acronym as the paper does before it starts to use it.

File: _summaries/maas2013rectifier_1.md [hunk 2]
```diff
@@ -3,17 +3,45 @@ layout: summary
...
+### Motivation for Rectifier Functions
+Drawbacks of sigmoidal activation functions:
+- Sigmoidal activation functions suffer from "vanishing gradients" which can slow training and/or cause the network to converge to poor local minima.
+- They do not induce hard-sparsity (values of 0) in activations, which can be a drawback if you want to use activations as feature representations for some downstream task.
+Both of these issues can, in principle, be solved with rectifier activation functions.
+
+<img width="500px" src="maas2013rectifier_1a.png"/>
+
+### Experiment setup
+- Task: Large Vocabulary Continuous Speech Recognition (LVCSR)
```
Comment (saikm200022) [inline _summaries/maas2013rectifier_1.md:26]: I would mention the type of loss function used and the type of optimization algorithm. Another part of their experiment was varying the number of nonlinear layers they used. I would also include that aspect of their experiment in your experiment setup.

File: _summaries/maas2013rectifier_1.md [hunk 3]
```diff
@@ -3,17 +3,45 @@ layout: summary
...
+- Data: 300-hour Switchboard conversational corpus
+- Model: DNN with ReLU, Leaky-ReLU, and tanh activation function
+- Evaluation metric: word-error rate (WER)
+- Side note: the authors validate that their baseline implementations are comparable to existing ones.
+
+### Results
+- Rectifier DNNs produce WER reductions of up to 2% as compared with sigmoidal DNNs, which is a substantial improvement for this task.
+- Rectifier DNNs benefit more from depth as compared with sigmoidal DNNs. Each time a hidden layer was added, rectifier DNNs show a greater absolute reduction in WER than sigmoidal DNNs.
```
Comment (saikm200022) [inline _summaries/maas2013rectifier_1.md:34]: It may be a good idea to mention that rectifier DNNs are less susceptible to vanishing gradient which explains why adding more hidden layers results in better performance when compared to sigmoidal DNNs.

File: _summaries/maas2013rectifier_1.md [hunk 4]
```diff
@@ -3,17 +3,45 @@ layout: summary
...
+- Choice of Rectifier function (leaky vs. non-leaky) does not impact performance. Leaky converges slightly faster.
+
+### Analyzing coding properties
+- ReLU DNNs contain substantially more sparse representations than sigmoidal DNNs.
+- ReLU networks, as compared with tanh networks, produce sparse codes where information is distributed more uniformly across hidden units.
```
Comment (saikm200022) [inline _summaries/maas2013rectifier_1.md:39]: I would define and mention the activation probability metric as well because that can reinforce your point about the sparsity of codes in ReLU DNNs.

File: _summaries/maas2013rectifier_1.md [hunk 5]
```diff
@@ -3,17 +3,45 @@ layout: summary
...
+- Evaluation metric: word-error rate (WER)
+- Side note: the authors validate that their baseline implementations are comparable to existing ones.
+
+### Results
+- Rectifier DNNs produce WER reductions of up to 2% as compared with sigmoidal DNNs, which is a substantial improvement for this task.
+- Rectifier DNNs benefit more from depth as compared with sigmoidal DNNs. Each time a hidden layer was added, rectifier DNNs show a greater absolute reduction in WER than sigmoidal DNNs.
+- Choice of Rectifier function (leaky vs. non-leaky) does not impact performance. Leaky converges slightly faster.
+
+### Analyzing coding properties
+- ReLU DNNs contain substantially more sparse representations than sigmoidal DNNs.
```
Comment (saikm200022) [inline _summaries/maas2013rectifier_1.md:38]: It might be worth it to mention that ReLU DNNs met the established benchmarks for the Switchboard task (can bring some context to how the tested models fair against established models).

**Review by saikm200022 — COMMENTED** [review]
Nice and concise summary! I have a few suggestions.
